signer-service/src/constants/tokenConfig.js

```diff
@@ -6,6 +6,14 @@ const TOKEN_CONFIG = {
     vaultAccountId: '6bsD97dS8ZyomMmp1DLCnCtx25oABtf19dypQKdZe6FBQXSm',
     minWithdrawalAmount: '10000000000000',
     maximumSubsidyAmountRaw: '1000000000000', // 1 unit
+    pendulumCurrencyId: {
```

Comment:
We could also avoid error-prone duplication here and determine this from `assetCode` and `assetIssuer` through some helper function – particularly once we add more currencies:

```js
      Stellar: {
        AlphaNum4: {
          code: `0x${Buffer.from(assetCode.padEnd(4, "\0")).toString("hex")}`,
          issuer: `0x${StrKey.decodeEd25519PublicKey(assetIssuer).toString("hex")}`,
        },
      },
```

Reply:
Could be but I'm not so sure since across the repo we are generally using directly the `hex` representation (or both), probably since it's easier to get from the `assetRegistry`. If we add this change I would prefer to be consistent and change everywhere.

Reply:
No, it's fine. Let's just leave it as it is.